simplq/src/components/pages/Admin/AddMember.jsx

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-import { Collapse, ListItemIcon } from '@material-ui/core';
 import AddIcon from '@material-ui/icons/Add';
-import { ExpandLess, ExpandMore } from '@material-ui/icons';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemText from '@material-ui/core/ListItemText';
-import JoinQueueForm from '../Join/Form';
+// import JoinQueueForm from '../Join/Form';
+import styles from '../../../styles/adminPage.module.scss';
 
-export default (props) => {
-  const [open, setOpen] = React.useState(false);
+export default () => {
+  // const [open, setOpen] = React.useState(false);
   return (
     <>
-      <ListItem button onClick={() => setOpen(!open)}>
-        <ListItemIcon>
-          <AddIcon />
-        </ListItemIcon>
-        <ListItemText primary="Add Manually" />
-        {open ? <ExpandLess /> : <ExpandMore />}
-        <Collapse in={open} timeout="auto" unmountOnExit>
-          <JoinQueueForm
-            buttonName="Add"
-            queueId={props.queueId}
-            joinQueueHandler={props.joinQueueHandler}
-          />
-        </Collapse>
-      </ListItem>
```

Comment:
I think it's better to have our own simple form here for now. I suggest us duplicating the handle click code here, and then refactoring later.

Reply:
I'll add the form in another PR. I remember seeing the design somewhere, just have to find it. It'll end up being very different from this old code anyway.

Reply:
Right